yb-voyager/cmd/exportSchema.go

```diff
@@ -176,7 +175,11 @@ func exportSchema() error {
 		return fmt.Errorf("failed to apply migration assessment recommendation to the schema files: %w", err)
 	}
 
-	applyMergeConstraintsTransformations()
+	// continue after logging the error; since this transformation is only for performance improvement
+	err = applyMergeConstraintsTransformations()
+	if err != nil {
+		log.Errorf("failed to apply merge constraints transformation to the schema files: %v", err)
```

Comment:
Shall we warn instead of error?

Reply:
hmm yeah i think we can do that, since this transformation is for better UX not fixing any failure.